Upstream doesn't prepend ${prefix} to the libdir, incluedir etc., so packages
which try to use OpenEXR fail to include it because it tries to include
"-Iinclude/OpenEXR" and not "-I/usr/include/OpenEXR"
diff --git a/IlmBase/IlmBase.pc.in b/IlmBase/IlmBase.pc.in
index 45fc4de..b1ac8e8 100644
--- a/IlmBase/IlmBase.pc.in
+++ b/IlmBase/IlmBase.pc.in
@@ -4,9 +4,9 @@
 ##
 
 prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+exec_prefix=${prefix}/@exec_prefix@
+libdir=${prefix}/@libdir@
+includedir=${prefix}/@includedir@
 libsuffix=@LIB_SUFFIX_DASH@
 Name: IlmBase
 Description: Base math and exception libraries
diff --git a/OpenEXR/OpenEXR.pc.in b/OpenEXR/OpenEXR.pc.in
index cf72f88..34fa246 100644
--- a/OpenEXR/OpenEXR.pc.in
+++ b/OpenEXR/OpenEXR.pc.in
@@ -4,9 +4,9 @@
 ##
 
 prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+exec_prefix=${prefix}/@exec_prefix@
+libdir=${prefix}/@libdir@
+includedir=${prefix}/@includedir@
 OpenEXR_includedir=@includedir@/OpenEXR
 libsuffix=@LIB_SUFFIX_DASH@
 
diff --git a/PyIlmBase/PyIlmBase.pc.in b/PyIlmBase/PyIlmBase.pc.in
index 7caaf7b..c242e36 100644
--- a/PyIlmBase/PyIlmBase.pc.in
+++ b/PyIlmBase/PyIlmBase.pc.in
@@ -4,9 +4,9 @@
 ##
 
 prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+exec_prefix=${prefix}/@exec_prefix@
+libdir=${prefix}/@libdir@
+includedir=${prefix}/@includedir@
 PyIlmBase_includedir=@includedir@/OpenEXR
 
 Name: PyIlmBase
